src/pages/library/DataBrowserDetails.js

```diff
@@ -0,0 +1,77 @@
+import { Fragment } from 'react'
+import { div, h, h1 } from 'react-hyperscript-helpers'
+import FooterWrapper from 'src/components/FooterWrapper'
+import { centeredSpinner, icon } from 'src/components/icons'
+import { libraryTopMatter } from 'src/components/library-common'
+
+
+const activeTab = 'browse & explore'
+const styles = {
+  page: { padding: 20, marginTop: 15 }
+}
+
+const getSnapshot = id => {
+  return {
+    namespace: 'harry-potter',
+    name: 'Harry Potter',
+    created: '2020-01-07T18:25:28.340Z',
+    lastUpdated: '2020-01-07T18:25:28.340Z',
+    tags: {
+      itemsType: 'AttributeValue',
+      items: ['1000 Genomes', 'CMG', 'Open Access', 'Exome']
+    },
+    description: 'The boy who lived',
+    lowerName: 'harry potter',
+    lowerDescription: 'lower description',
+    project: {
+      id: '112',
+      name: 'CDC'
+    },
+    subjects: 10,
+    dataType: 'RNA Seq',
+    keepCollapsed: true,
+    locked: false,
+    files: 15,
+    fileSize: 4432
+  }
+}
+
+const DataBrowserDetails = routeParams => {
+  const id = routeParams.id
+  const snapshot = getSnapshot(id)
+
+  return h(FooterWrapper, { alwaysShow: true }, [
+    libraryTopMatter(activeTab),
+    !snapshot ?
+      centeredSpinner() :
+      h(Fragment, [
+        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'top', width: '100%' } }, [
+          div({ style: styles.page }, [
+            icon('angle-left', { size: 35 })
+          ]),
+          getMainContent(snapshot),
+          getSidebar(snapshot)
```

Comment:
```suggestion
          h(MainContent, { snapshot }),
          h(Sidebar)
```
We're dropping the input argument `snapshot` for the `Sidebar` call since it isn't used at the moment. See comment further down on `Sidebar` implementation.

Reply:
I have a follow up PR that will use the snapshot in the Sidebar call - https://github.com/DataBiosphere/terra-ui/pull/2623/files

Since this was a PR focused on just setting up the basics, I thought it would be fine for this to be here since it will be used.

Reply:
How can I tell the difference between when to minimize code? It seems like we shouldn't need to wrap these elements with the `h`, because it's rendering without being explicitly declared. The get-- methods are returning hyperscript objects that are rendering properly

Reply:
I can see how this can be confusing. The key idea is that react components conceptually are passed two parameters when they're created: 

1. Props - An object whose key/value pairs are the properties passed in to the component
2. Children - A list of child components that will be rendered by this component somewhere inside of it

You can create a function that returns a react component - which is what the `get` functions are doing. But since the intent of the code is to render something to the page (correct me if I am wrong), it would be best to define these functions as [react function components](https://reactjs.org/docs/hooks-state.html#hooks-and-function-components)

A react component needs to have a capitalized name, and should accept a list of props as a parameter - as a convention in this code base, we name this parameter `props`. Making this a function component will make it easy to add children to it later on and further enhance the functionality, for example adding state, later on without having to change anything.

**TL;DR** When in doubt, make a Component, rendered using `h`. This makes sure React will know what to do with it in any situation.

Reply:
Hey, I'm sorry, I wasnt able to make this change - This error popped up on this change

<img width="951" alt="Screen Shot 2021-09-15 at 5 18 51 PM" src="https://user-images.githubusercontent.com/10501914/133510940-a6a05bf6-d3d5-4ba5-9b26-73d59c167347.png">